tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -0,0 +1,78 @@
+/* bender-tags: inlinetoolbar */
+/* bender-ckeditor-plugins: toolbar,link,inlinetoolbar */
+/* bender-include: ../balloonpanel/_helpers/tools.js */
+/* global balloonTestsTools */
+
+( function() {
+	'use strict';
+
+	var restore = CKEDITOR.dom.window.prototype.getViewPaneSize;
+	CKEDITOR.dom.window.prototype.getViewPaneSize = function() {
```

Comment:
As @Comandeer suggested, please use `sinon.stub` and move it to `setUp` and `tearDown` methods. As we don't have `after` method, and only `tearDown` one (like `afterEach`) it makes sense to keep it in `setUp` (like `beforeEach`) and `tearDown` methods, so the stub should be created in `setUp` and then restored in `tearDown`.